examples/checkboxes-checked.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en" class="govuk-template">
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
```

Comment:
The meta tag on GOV.UK is simply `<meta name="viewport" content="width=device-width, initial-scale=1">` - is there a reason why we want `viewport-fit=cover` too?

Reply:
I added this based on the GOV.UK template file but for this specific example, it's not needed indeed so I removed it.

Reply:
Doesn't matter either way IMO - I'm more interested in why the same meta value hasn't been applied to the homepage :)

Reply:
The GOV.UK homepage uses a deprecated layout at the moment. Shall be updated – whenever resources are dedicated to this.